src/exchanges/fetch.ts

```diff
@@ -146,3 +151,24 @@ const executeFetch = (
       }
     });
 };
+
+export const convertToGet = (uri: string, body: Body): string => {
+  const queryParams: string[] = [];
+  if (body.query) {
+    queryParams.push(`query=${encodeURIComponent(body.query)}`);
+  }
+
+  if (body.operationName) {
+    queryParams.push(`operationName=${encodeURIComponent(body.query)}`);
```

Comment:
This isn’t correct afaik. This is usually the query’s name that is to be executed, but should be optional when only one `DocumentNode` is present (so for us all the time)

Since we’re not adding this for `POST` we shouldn’t add it to `GET` or we should add it for both.